### PR TITLE
Prepare release 5.0.0

### PR DESCRIPTION
**Pull request by abikouo**
Release `5.0.0`

### CODE REVIEW

**Comment by abikouo** [comment]
recheck

**Review by gravesm — COMMENTED** [review]
Can you make the changes to deprecations and required ansible version on the main branch and backport?

**Comment by abikouo** [comment]
> Can you make the changes to deprecations and required ansible version on the main branch and backport?

@gravesm here is the pull request
https://github.com/ansible-collections/kubernetes.core/pull/733

**Review by gravesm — APPROVED** [review]

**Review by mandar242 — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,24 @@ Kubernetes Collection Release Notes
...
 
+v5.0.0
+======
+
+Release Summary
+---------------
+
+This major release drops support for ``ansible-core<2.15``.
+
+Minor Changes
```
Comment (yurnov) [inline CHANGELOG.rst:15]: somehow, #744 is not included in the changelog. Assuming that the reason for that is that PR prepared earlier than #741 merged and this PR wasn't updated

**Comment by yurnov** [comment]
Hi @abikouo,

could you please also prepare release 3.2.0 from `stable-3` branch? I gave a explanations of that [here](https://github.com/ansible-collections/kubernetes.core/pull/741#issuecomment-2144739041), shortly, Ansible 10.x will include only 3.x.y versions of `kubernetes.core` collection and recent [documentation change](https://github.com/ansible-collections/kubernetes.core/pull/741) appearing on [Ansible site](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/kubectl_connection.html#ansible-collections-kubernetes-core-kubectl-connection) require a new 3.x.x version or a next major version of Ansible (that will be Ansible 11).

Also, I can prepare PR for the 3.2 from my side.

**Pull request merged**